src/tests/helpers.js

```diff
@@ -240,7 +240,11 @@ export const mockSelectInputSolo = {
 
 export const mockSelectInputSoloCreatable = {
   ...mockSelectInputSolo,
-  oneOf: [...mockSelectInputSolo.oneOf, { type: 'string', title: '{Create another}' }],
+  creatable_on: 'browser_name',
```

Comment:
question: I don't understand, why do we need this new keyword in json-schema-form? How does it influence the mechanisms of `json-schema-form` itself?

If I'm understanding correctly, this MR is just about removing the select support for `type: "string"` because we no longer need it internally. Or am I missing something?

Reply:
@sandrina-p sorry, makes total sense. Let me update it and tag you as a reviewer

Reply:
@joeynimu you are logged with Remotecom account, but you should be using your own account :)

Reply:
yikes!

Reply:
Okay the code is coode now! The missing part is to update the PR title and description to reflect what was really done. Once you do that, I'll approve the PR :)

Reply:
@sandrina-p took me a while to come back to this, I can blame PTO + other priorities but I have updated the title and description. Let me know if this works.

Reply:
Thanks! The correct naming though is `fix(select): Remove creatable logic". The select makes it explicit this is for select fields. "Remove" make it explicit that we removed a unwanted feature, "clean up" sound more like tyding/refactor.

Approved :)

Reply:
To make it clear, this can still be useful in upcoming cases, but as it's not needed right now, better to delete it. We can always add it back later when needed.

Reply:
@sandrina-p noted and updated ✅ thanks for the review